Makefile

```diff
@@ -1,9 +1,8 @@
 PYTHON=`which python`
-PYTHON2=`which python2`
 PYTHON3=`which python3`
```

Comment:
Should `PYTHON` be removed as well and replaced by `PYTHON3`? Or is there a reason to use the default python version to build the source and install targets?

Reply:
Possibly related: all the source files have a `#!/usr/bin/python` at the top. Should these be changed to refer to python3? Or maybe they should be removed?